config_defaults/subtests/docker_cli/systemd.ini

```diff
@@ -1,8 +1,17 @@
 [docker_cli/systemd]
-docker_timeout = 300
+subsubtests = systemd_pull, systemd_build , systemd_run
+
 #: Systemd unit file
+sysd_unit_file = docker-pull.service
+#: image name to pull, build, run
+image_name = rhel7:latest
```

Comment:
This could break on non-RHEL systems (i.e. Fedora or CentOS) depending on how subscriptions/entitlement enforcement is worked out.  This is a problem for fixing on another day.